docs/src/main/sphinx/connector/influxdb.rst

```diff
@@ -0,0 +1,197 @@
+
+Requirements
+------------
+
+To connect to InfluxDB, you need:
+
+* InfluxDB 1.x or higher.
+* Network access from the Trino coordinator and workers to InfluxDB.
+
+Configuration
+-------------
+
+To configure the InfluxDB connector, create a catalog properties file
+``etc/catalog/influxdb.properties`` with the following contents,
+replacing the properties as appropriate:
+
+.. code-block:: properties
+
+    connector.name=influxdb
+    influx.host=localhost
+    influx.port=8086
+    influx.protocol=http
+    influx.username=username
+    influx.password=password
+
+Configuration properties
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. list-table:: The following configuration properties are available
+    :widths: 35, 55, 10
+    :header-rows: 1
+
+    * - Property name
+      - Description
+      - Default
+    * - ``influx.host``
+      - Hostname of the InfluxDB server to connect to. This property is required.
+      -
+    * - ``influx.port``
+      - Port of the InfluxDB server to connect to. This property is required.
+      -
+    * - ``influx.protocol``
+      - Protocol of the InfluxDB server to connect to.
+      - ``http``
+    * - ``influx.username``
+      - User name to use to connect to InfluxDB.
+      -
+    * - ``influx.password``
+      - Password name to use to connect to InfluxDB.
+      -
+    * - ``influx.cacheTimeOut``
+      - How long for meta data (table schemas、columns schemas) to cache in memory.
+      - ``Never``
+    * - ``influx.connectTimeOut``
+      - The socket connect timeout to InfluxDB server.
+      - ``10s``
+    * - ``influx.writeTimeOut``
+      - The socket write timeout to InfluxDB server.
+      - ``10s``
+    * - ``influx.readTimeOut``
+      - The socket read timeout to InfluxDB server.
+      - ``10s``
+    * - ``influx.optimize.filterPushDownEnable``
+      - Set true to enable filter pushdown.
+      - ``false``
+    * - ``influx.optimize.limitPushDownEnable``
+      - Set true to enable limit pushdown.
+      - ``false``
+    * - ``influx.optimize.projectionPushDownEnable``
+      - Set true to enable projection pushdown.
+      - ``false``
+    * - ``influx.aggregationPushDownEnable``
+      - Set true to enable aggregation pushdown.
+      - ``false``
+
+.. _influxdb-type-mapping:
+
+Type mapping
+------------
+
+Because Trino and InfluxDB each support types that the other does not, this
+connector :ref:`maps some types <type-mapping-overview>` when reading data.
+
+
+InfluxDB type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps InfluxDB types to the corresponding Trino types
+according to the following table:
+
+.. list-table:: InfluxDB type to Trino type mapping
+  :widths: 30, 30, 50
+  :header-rows: 1
+
+  * - InfluxDB type
+    - Trino type
+    - Notes
+  * - ``BOOLEAN``
+    - ``BOOLEAN``
+    -
+  * - ``FLOAT``
+    - ``DOUBLE``
+    -
+  * - ``INTEGER``
+    - ``BIGINT``
```

Comment:
Why did you map `INTEGER` to `BIGINT`?

Reply:
integer is int64 in InfluxDB data type actually,  same with float.
so i map integer to bigint  and float to double
https://docs.influxdata.com/influxdb/v1.8/write_protocols/line_protocol_reference/#data-types

Reply:
Thanks, let's mention it in the `Notes` column.